test/svg-test-helper.js

```diff
@@ -80,3 +82,88 @@ export const calculateD3Path = (props, pathType, index = 0) => {
 
   return undefined;
 };
+
+export const parseSvgPathCommands = (commandStr) => {
+  const matches = commandStr.match(
+    /[MmLlHhVvCcSsQqTtAaZz]+[^MmLlHhVvCcSsQqTtAaZz]*/g
+  );
+
+  return matches.map((match) => {
+    const name = match.charAt(0);
+    const args = match
+      .substring(1)
+      .split(",")
+      .map((arg) => {
+        return parseFloat(arg, 10);
+      });
+
+    return {
+      raw: match,
+      name,
+      args
+    };
+  });
+};
+
+export const getPathCommandsFromContainer = (container) => {
+  const commandStr = container.querySelector("path").getAttribute("d");
+  return parseSvgPathCommands(commandStr);
+};
+
+const exhibitsShapeSequence = (commandString, shapeSeqeuence) => {
+  const commands = parseSvgPathCommands(commandString);
+  return commands.every(
+    (command, index) => command.name === shapeSeqeuence[index]
+  );
+};
+
+export const svgExpectations = {
+  /**
+   * Asserts that the provided command string produces a rectangular shape.
+   *
+   * @param {string} commandString - the "d" attribute of a `path` element.
+   * @returns {void}
+   */
+  expectIsRectangular(commandString) {
+    const isBar = exhibitsShapeSequence(commandString, RECTANGULAR_SEQUENCE);
```

Comment:
As noted above, let's just export `isBar` here instead of including the test expecation.